README_LD.md

```diff
@@ -0,0 +1,74 @@
+Takeaway Challenge
+==================
+```
+                            _________
+              r==           |       |
+           _  //            |  M.A. |   ))))
+          |_)//(''''':      |       |
+            //  \_____:_____.-------D     )))))
+           //   | ===  |   /        \
+       .:'//.   \ \=|   \ /  .:'':./    )))))
+      :' // ':   \ \ ''..'--:'-.. ':
+      '. '' .'    \:.....:--'.-'' .'
+       ':..:'                ':..:'
+
+ ```
+
+Summary
+-----
+
+**The Restaurant**
+* Dish
+  * A dish is created with a name and a prince. 
+  * Prices are input as a float with two decimal points.
+  * Dishes can be itemised as a hash (itemise method).
+
+* Menu
+  * Dishes (itemised as a hash) can be added to the menu (add_to_menu(dish) method)
+  * Menu can be viewed (view_menu method)
+  * DEFAULT_MENU constant supplied, access with access_menu method
+  * Added a banner for the takeaway cafe: "*** TDD Greasy Spoon ***"
+
+**The Customer**
+* Order
+  * An order is initialized with an empty my_order array 
+  * Dishes can be added one by one to the order, receives name and quantity as arguments (add_dish_to_order(name, quantity) method)
+  * Cost of order calculated (order_total method)
+  * Added view_order method for customer which includes cost of item 
+  * Added how_much method to output total order cost
+
+
+Notes
+-----
+
+ * Disappointed I couldn't figure out Twilio. It was asking me to pay for the number?
```

Comment:
When you signed up to the free trial it gives you credit of around £11, you can use this to 'buy' a twilio number for about 80p a month - so doesn't actually charge you! You then use this as the number you send the sms message from.